Method listing corrected to include "delete"

richard-fairthorne opened the pull request:
The method listing incorrectly included post twice, and did not include delete. This change fixes that.

dsternlicht reviewed (changes requested):
I'm not sure I understand. Why did you removed the `post` method?

richard-fairthorne commented:
![Screen Shot 2022-01-04 at 12 03 59 PM](https://user-images.githubusercontent.com/1225460/148096202-6455b95f-afa1-4199-9993-9fd56704c0bc.png)

It is in the list twice. I removed the duplicate, and added the missing "delete" method.
